### PR TITLE
Feature/gallery

### PR DESCRIPTION
**Pull request by kadjialan**
Design of a Figma template  using reactjs

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-      <Home />
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route index path="/" element={<Home />} />
+        <Route path="/about" element={<About />} />
+        <Route path="/tourpackage" element={<Package />} />
+        <Route path="/gallery" element={<Gallery />} />
+        <Route path="/contact" element={<Contact />} />
+        <Route path="*" element={<Blog />} />
```
Comment (gastonche) [inline src/App.jsx]: The component here should not be blog. It should be a generic fallback component. So rename it to something more meaningful

File: src/Pages/About/Hero2/Hero2.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Searchnav from '../../../components/Searchnav/Searchnav';
+import Monkey from '../../../asserts/images/monkey.png';
+import icon from '../../../asserts/images/icon.png';
+import './Hero2.css';
+
+export default function Hero2() {
```
Comment (gastonche) [inline src/Pages/About/Hero2/Hero2.jsx]: Hero2 is not an appropriate name for a react component. Make the name more meaningful. Is it not AboutPageHeroSection?

File: src/Pages/About/Safarri/Safarri.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.safarri__cards {
```
Comment (gastonche) [inline src/Pages/About/Safarri/Safarri.css:1]: Nice going with the implementation of BEM here.

File: src/Pages/About/Safarri/Safarri.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Saf1 from '../../../asserts/images/saf1.png';
```
Comment (gastonche) [inline src/Pages/About/Safarri/Safarri.jsx]: When importing files you want to give them the right names. Saf1, Saf2, Saf3 are all not valid here

File: src/Pages/About/Safarri/Safarri.jsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+import Saf3 from '../../../asserts/images/saf3.png';
+import './Safarri.css';
+
+export default function Safarri() {
+  return (
+    <div className="safarri">
+      <h1 className="xaf">Welcome to Africa Wizzy Safari</h1>
+      <hr />
+      <div className="safarri__holder">
+        <div className="safarri__cards">
```
Comment (gastonche) [inline src/Pages/About/Safarri/Safarri.jsx:13]: This section can be broken down into sub components. To make it more readable. but for now this is acceptable.

File: src/Pages/About/aaa.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+/* import React, { useEffect } from 'react'; */
```
Comment (gastonche) [inline src/Pages/About/aaa.jsx]: This file seems useless. Please remove it.

File: src/Pages/Blog/Blog.jsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Blog.css';
+
+export default function Blog() {
```
Comment (gastonche) [inline src/Pages/Blog/Blog.jsx]: This component should probably be NotFound instead*

File: src/Pages/Contact/Contact.jsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+import Searchnav from '../../components/Searchnav/Searchnav';
+import Footer from '../Home/Footer/Footer';
+import Address from './Address/Address';
+
+export default function Contact() {
+  return (
+    <div>
+      <Navbar />
+      <Searchnav />
+      <Hero3 data="Contact Us" text="Gallery" />
```
Comment (gastonche) [inline src/Pages/Contact/Contact.jsx]: Should text here be Gallery?

File: src/Pages/Gallery/Pictures/Picture.jsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Pic1 from '../../../asserts/images/folder/pic1.png';
```
Comment (gastonche) [inline src/Pages/Gallery/Pictures/Picture.jsx]: Again, we need meaningful names for file we import.

File: src/Pages/Gallery/Pictures/Picture.jsx [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+export default function Picture() {
+  return (
+    <div className="pictures">
+      <div className="pictures__btn">
+        <Buttons content="Open Instagram" />
+      </div>
+      <div className="pictures__content">
+        <div className="pictures__content__one">
+          <img src={Pic1} alt="afrique" className="pic" />
```
Comment (gastonche) [inline src/Pages/Gallery/Pictures/Picture.jsx]: Is there a reason why you are not looping here?

File: src/components/Buttons/Buttons.jsx [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Buttons.css';
+import PropTypes from 'prop-types';
+
+export default function Buttons({ content }) {
```
Comment (gastonche) [inline src/components/Buttons/Buttons.jsx]: This component should be called Button not Buttons. It displays one Button right?

File: src/components/Hero3/Hero3.jsx [hunk 12]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Play from '../../asserts/images/play.png';
+import './Hero3.css';
+
+export default function Hero3({ data, text }) {
```
Comment (gastonche) [inline src/components/Hero3/Hero3.jsx]: This prop called data here, don't you think it should rather be called title?

File: src/components/Searchnav/Searchnav.jsx [hunk 13]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Africa from '../../asserts/images/Africa.png';
+import Buttons from '../Buttons/Buttons';
 import './Searchnav.css';
 
 export default function Searchnav() {
```
Comment (gastonche) [inline src/components/Searchnav/Searchnav.jsx]: This component is call Searchnav. 

- Firstly it is not in camel case
- Also don't you think it should be named something else. Like is it really a "Search Nav bar"?

**Review by gastonche — CHANGES_REQUESTED** [review]
Take a look at these comments and handle them

**Comment by jordan-ae** [comment]
Fix Linter errors

**Pull request closed without merging**